lib/rubocop/cop/rspec/describe_method.rb

```diff
@@ -23,20 +23,25 @@ class DescribeMethod < Base
         MSG = 'The second argument to describe should be the method ' \
               "being tested. '#instance' or '.class'."
 
-        # @!method second_argument(node)
-        def_node_matcher :second_argument, <<~PATTERN
+        # @!method describe_method(node)
+        def_node_matcher :describe_method, <<~PATTERN
           (block
-            (send #rspec? :describe _first_argument $(str _) ...) ...
-          )
+            (send #rspec? :describe _first_argument
+              ${(str #method_name?) (dstr (str #method_name?) ...)}
+            ...)
+          ...)
         PATTERN
 
         def on_top_level_group(node)
-          second_argument = second_argument(node)
+          describe_method(node) do |describe|
+            add_offense(describe)
+          end
+        end
 
-          return unless second_argument
-          return if second_argument.str_content.start_with?('#', '.')
+        private
 
-          add_offense(second_argument)
+        def method_name?(description)
```

Comment:
This seems to do the opposite, like "not_a_method_name".

Reply:
Oops, thanks for catching this! I refactored and organized it. What do you think?